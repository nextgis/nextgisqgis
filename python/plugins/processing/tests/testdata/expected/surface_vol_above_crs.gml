<?xml version="1.0" encoding="utf-8" ?>
<ogr:FeatureCollection
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://ogr.maptools.org/ surface_vol_above_crs.xsd"
     xmlns:ogr="http://ogr.maptools.org/"
     xmlns:gml="http://www.opengis.net/gml">
  <gml:boundedBy><gml:null>missing</gml:null></gml:boundedBy>
                                                                                                                                                                                                                                                                                                
  <gml:featureMember>
    <ogr:surface_vol_above_crs fid="surface_vol_above_crs.0">
      <ogr:volume>413784918.00990051</ogr:volume>
      <ogr:m2>11497732.58555590</ogr:m2>
      <ogr:pixel_count>64692</ogr:pixel_count>
    </ogr:surface_vol_above_crs>
  </gml:featureMember>
</ogr:FeatureCollection>
