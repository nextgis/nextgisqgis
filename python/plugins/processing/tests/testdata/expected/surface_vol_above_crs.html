<html><head><meta http-equiv="Content-Type" content="text/html;charset=utf-8"/></head><body>
<p>Analyzed file: /home/nyall/dev/QGIS/python/plugins/processing/tests/testdata/custom/dem_crs.tif (band 1)</p>
<p>Volume: 413784918.0099005</p>
<p>Pixel count: 64692</p>
<p>Area: 11497732.5855559 m2</p>
</body></html>